src/main/java/uk/gov/hmcts/ccd/CoreCaseDataApplication.java

```diff
@@ -31,4 +43,26 @@ public static void main(String[] args) {
         SpringApplication.run(CoreCaseDataApplication.class, args);
     }
 
+    @Bean
+    public Executor asyncExecutor() {
+        //thread pool used for the execution of async tasks (methods marked with @Async)
+        ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+        executor.setCorePoolSize(applicationParams.getAsyncCorePoolSize());
+        executor.setMaxPoolSize(applicationParams.getAsyncMaxPoolSize());
+        executor.setQueueCapacity(500);
```

Comment:
Should this not be a applicationParams config item?

Reply:
don't think it's necessary for everything to be part of applicationParams. I picked the settings that are likely to change more frequently. queue capacity is not one of them